cmd/bootstrap/runner.go

```diff
@@ -8,8 +8,8 @@ import (
 	"time"
 
 	"github.com/giantswarm/apiextensions/v3/pkg/apis/application/v1alpha1"
-	"github.com/giantswarm/apiextensions/v3/pkg/crd"
 	"github.com/giantswarm/apiextensions/v3/pkg/label"
+	crdpkg "github.com/giantswarm/app/v5/pkg/crd"
```

Comment:
```suggestion
	pkgcrd "github.com/giantswarm/app/v5/pkg/crd"
```

Just to use our usual format for aliases.